developerportal/templates/molecules/card-featured-primary.html

```diff
@@ -3,7 +3,14 @@
 {% load wagtailimages_tags %}
 {% image resource.image width-1024 as image %}
 {% image resource.card_image width-1024 as card_image %}
-{% static "img/placeholders/article.jpg" as fallback_image_url %}
+
+{% if block_type == "video" %}
+  {% static "img/placeholders/video.jpg" as fallback_image_url %}
+{% elif block_type == "event" %}
+  {% static "img/placeholders/event.jpg" as fallback_image_url %}
+{% else %}
+  {% static "img/placeholders/article.jpg" as fallback_image_url %}
+{% endif %}
```

Comment:
While adding support for a video-specific fallback thumbnail, i noticed that there was no Event-specific one either, so I added that at the same time